src/Scribunto/ScribuntoLuaLibrary.php

```diff
@@ -33,6 +33,28 @@ public function register(): void {
 		$this->getEngine()->registerInterface( __DIR__ . '/' . 'mw.wsslots.lua', $interfaceFuncs, [] );
 	}
 
+	/**
+	 * @param \WikiPage $wikiPage
+	 *
+	 * @return bool
+	 */
+	private function userCan( \WikiPage $wikiPage ): bool {
```

Comment:
Use a `use` statement to import WikiPage and omit the global namespace prefix (`\`).